android/build.gradle

```diff
@@ -83,6 +90,7 @@ android {
   }
 
   packagingOptions {
+    doNotStrip resolveBuildType() == 'debug' ? "**/**/*.so" : ''
```

Comment:
This should prevent stripping of debug symbols while not in release mode. I'm guessing it will shave disk space, but more importantly, I wonder if there is some sort of performance penalty for not stripping the symbols.